src/main/java/io/cdap/e2e/pages/actions/CdfStudioActions.java

```diff
@@ -166,11 +337,28 @@ public static void sinkGcs() {
     CdfStudioLocators.gcs.click();
   }
 
-  public static void clickPreviewConfigRunButton() {
-    CdfStudioLocators.previewConfigRunButton.click();
+  /**
+   * Click on the Validate button inside Plugin's properties page
+   *
+   * @deprecated Use {@link CdfPluginPropertiesActions#clickValidateButton()}
+   */
+  @Deprecated
+  public static void clickValidateButton() {
```

Comment:
call the new method inside the old method instead of changing the implementation of old method.
Also can we create a JIRA for cleaning up all the deprecated methods in all the plugin specific PRs at once?
Can we also include the list of deprecated methods that are needed to be replaced in the JIRA?

Reply:
1. Updated it to use the new method.
2. JIRA created to clean up the deprecated methods in the Plugin specific PR: https://cdap.atlassian.net/browse/CDAP-18909
3. Will do and add the details in the above mentioned ticket.